pandas/core/indexes/base.py

```diff
@@ -1552,12 +1552,19 @@ def droplevel(self, level=0):
 
         levnums = sorted(self._get_level_number(lev) for lev in level)[::-1]
 
-        if len(level) == 0:
+        return self._drop_level_numbers(levnums)
+
+    def _drop_level_numbers(self, levnums: List[int]):
```

Comment:
sorry to be pedantic, can you make this _drop_level_number (everything else is singular)

Reply:
it specifically takes `List[int]`